dashboard/test/controllers/sections_controller_test.rb

```diff
@@ -16,6 +16,9 @@ class SectionsControllerTest < ActionController::TestCase
 
     @flappy_section = create(:section, user: @teacher, login_type: 'word', script_id: Unit.flappy_unit.id)
     @flappy_user_1 = create(:follower, section: @flappy_section).student_user
+
+    @sections = create_list :section, 3, user: @teacher
+    @section = @sections.first
```

Comment:
I don't think these are needed?

Reply:
Yes, that is correct.  I removed them and re-tested just to check.